src/paho/mqtt/client.py

```diff
@@ -2671,6 +2694,83 @@ def _reconnect_wait(self):
             time.sleep(min(remaining, 1))
             remaining = target_time - time_func()
 
+    def _get_proxy(self):
+        def proxy_is_valid(p):
+            if isinstance(p, dict):
+                return bool(p.get("proxy_type"))
+            elif isinstance(p, (list, tuple)):
```

Comment:
Why is list/tuple allowed?

Reply:
PySocks' `socks.get_default_proxy()` method returns the configured proxy as a tuple. Also, I might not be understanding properly, but how would `**` unpack a list/tuple?

Reply:
I think I meant `*`.